src/fsharp/vs/Symbols.fs

```diff
@@ -194,7 +196,9 @@ and FSharpEntity(cenv:cenv, entity:EntityRef) =
     let isResolvedAndFSharp() = 
         match entity with
         | ERefNonLocal(NonLocalEntityRef(ccu, _)) -> not ccu.IsUnresolvedReference && ccu.IsFSharp
-        | _ -> true
+        | _ -> 
+            let ccu = defaultArg (ItemDescriptionsImpl.ccuOfItem cenv.g (getItem entity)) cenv.thisCcu 
+            ccu.IsFSharp
```

Comment:
I think these three lines should just be a single line that changes the "true" case on line 197 to "cenv.thisCcu.IsFSharp".  I think the getItem change is then no longer needed and the whole fix becomes one line.